src/SearchResults.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import moment from "moment";
+
+function SearchResults(props) {
+  return (
+    <div>
+      <table className="table">
+        <thead>
+          <tr>
+            <th scope="col">Id</th>
+            <th scope="col">Title</th>
+            <th scope="col">First Name</th>
+            <th scope="col">Last Name</th>
+            <th scope="col">Email</th>
+            <th scope="col">Room Id</th>
+            <th scope="col">Check in Date</th>
+            <th scope="col">Check out Date</th>
+            <th scope="col">No of Nigths</th>
```

Comment:
You could potentially generate these <th> elements dynamically with a map function. It would save you some lines of code. Let me know if you're not sure what I mean!